scripts/combine/setupCombine.py

```diff
@@ -783,14 +775,21 @@ def assertSample(name, startsWith=["W", "Z"], excludeMatch=[]):
             )
 
         if args.fakeSmoothingMode == "full":
-            # add nominal nonclosure of QCD MC as single systematic
+            # add systematic of explicit parameter variation
             def fake_nonclosure(h, axesToDecorrNames, *args, **kwargs):
-                # apply QCD MC nonclosure by adding parameters (assumes log space, e.g. in full smoothing)
-                fakeselector.external_params = params_d - params
+                # apply varyation by adding parameter value (assumes log space, e.g. in full smoothing)
+                fakeselector.external_params = np.zeros(4)
+                fakeselector.external_params[1] = 0.5
                 hvar = fakeselector.get_hist(h, *args, **kwargs)
                 # reset external parameters
                 fakeselector.external_params = None
 
+                hnom = fakeselector.get_hist(h, *args, **kwargs)
+
+                # normalize variation histogram to have the same integral as nominal
+                scale = hnom.sum(flow=True).value / hvar.sum(flow=True).value
+                hvar = hh.scaleHist(hvar, scale)
```

Comment:
Do you want to do this by normalizing the full histogram, or would it make more sense to normalize seperately in each eta-charge bin?

Reply:
OK this makes only small differences but I agree and it's done